torchft/process_group.py

```diff
@@ -573,8 +553,8 @@ class _BabyWork(Work):
     def __init__(
         self,
         pg: "ProcessGroupBaby",
-        tx: mp.Queue,
-        rx: mp.Queue,
+        tx: _MonitoredQueue,
```

Comment:
What are `tx` and `rx` meant to represent?

Reply:
These are the raw communication channels with the subprocess, might be good to refactor these so all queue comms go through the PG

Reply:
refactored